### ISSUE TITLE
ESLint warning on class in dedicated file

### ISSUE THREAD

**Issue by dwaard** [HZ-HBO-ICT/ts-skeleton-app#12]
If a class is declared in its own dedicated file (i.e. `Game` in `game.ts`), ESLint-typescript raises a warning:

Proposed solution
Add a rule to ignore `no-unused-vars`:
```
    // This setting is switched off to prevent warning on classes in dedicated files
    '@typescript-eslint/no-unused-vars': 'off',
```

**Comment by dwaard** [comment]
DISCUSSION: an alternative solution might be to export/import classes as modules. 
Potential benefits for students:
- Possibly no need to/replaces `/// <reference path="Game.ts"/>`
- Is a bit like the `use` statement in PHP

Needed to implement:
- add `"module": "AMD",` to tsconfig.json
- add `export default` before each class
- add `import Game from './Game';` to import classes when needed

**Comment by FransBlauw** [comment]
Fixed by #16
